rules/catch-error-name.js

```diff
@@ -38,6 +38,9 @@ function indexifyName(name, scope) {
 const create = context => {
 	const opts = context.options[0];
 	const name = (opts && opts.name) || 'err';
```

Comment:
Use Object.assign so you don't have to check whether options exist or not each time:

```js
const opts = Object.assign({}, {name: 'err'}, context,options[0]);
```